src/mpc/inputs.jl

```diff
@@ -135,6 +192,18 @@ function setup_pv_inputs(s::MPCScenario, existing_sizes, production_factor)
     return nothing
 end
 
+function setup_wind_inputs(s::MPCScenario, existing_sizes, production_factor)
```

Comment:
In MPC versions of setup_X_inputs()
Why don’t we need to do fillin_techs_by_exportbin() or create om_cost_per_kw like we do in the non MPC version of these functions?